content/guides/monitors.md

```diff
@@ -143,6 +157,12 @@ on the left. This guide will walk through the configuration of the Metric type.
         *time aggregation* and a *time window* on which the change will be
         calculated.
 
+    - For details on how to configure Anomaly Detection please visit the
```

Comment:
Nits: omit please; s/visit/see

For details on how to configure Anomaly Detection, see the Anomaly Detection Guide